dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/work/WorkRepositoryImpl.java

```diff
@@ -66,13 +65,9 @@ public static String getSqlUpdateFieldValues() {
 
   @Autowired
   public WorkRepositoryImpl(
-      Jdbi dbi,
-      IdentifierRepository identifierRepository,
-      AgentRepositoryImpl agentRepositoryImpl,
-      ItemRepositoryImpl itemRepositoryImpl) {
+      Jdbi dbi, AgentRepositoryImpl agentRepositoryImpl, ItemRepositoryImpl itemRepositoryImpl) {
```

Comment:
Hier wird nach wie vor ein Repository verwendet, gehört wohl eher in den Service, oder?

Reply:
Wird nur für Select gebraucht. Könnte also hier bleiben.

Reply:
Ok :+1: